waterlog/src/Pages/Usage.js

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from "react";
 import Usage from "./../Components/Usage";
+import TryCedric from "./../Components/TryCedric";
```

Comment:
What is this component? and Why is it a developer's name

Reply:
Removed dummy component.